README.md

```diff
@@ -17,6 +21,15 @@ born to bring that same vision to the other platforms supported by Ebitengine.
 - **Dynamic Linking**: Load symbols at runtime and use it as a plugin system.
 - **Foreign Function Interface**: Call into other languages that are compiled into shared objects.
 
+## Supported Platforms
+
+- **FreeBSD**: amd64, arm64
+- **Linux**: amd64, arm64
+- **macOS**: amd64, arm64
```

Comment:
Sorry I forgot about iOS.

`macOS/iOS`?

Reply:
oh true.... How did I forget that!